lib/IDE/SyntaxModel.cpp

```diff
@@ -1318,6 +1315,7 @@ bool ModelASTWalker::findUrlStartingLoc(StringRef Text,
       return true;
     }
   }
+#endif
   return false;
```

Comment:
I patched it both ways but settled on this way because it looked cleaner. I'll change it back I guess.